tests/integration/README.md

```diff
@@ -0,0 +1,39 @@
+# Prerequisites
+
+- Machine with NVIDIA GPU
+- Operation system Linux
+- [Golang >= 1.18 installed](https://golang.org/)
+- [DCGM installed](https://developer.nvidia.com/dcgm)
+
+# Integration Tests
+
+## Basics
+
+From the dcgm-exporter root directory run the tests:
+
+```
+make test-integration
+```
+
+## Quickly iterating on a single test
+
+Run a single test
+
+```
+make test-integration -e TEST_ARGS="-test.run [test name here]"
+```
+
+Example:
+```
+make test-integration -e TEST_ARGS="-test.run TestStartAndReadMetrics"
+```
+
+**WARRNING**: It takes about 30 seconds, before the dcgm-exporter instance will read available metrics.
```

Comment:
Some of the metrics require at least two data points to compute a value, which means at least one polling interval should be passed before we are able to get the results. By default, dcgm-exporter uses 30 seconds polling interval, thus the delay you observe.

Reply:
I updated the readme with this information. @nikkon-dev , thank you.